packages/rocketchat-ui/client/components/icon.js

```diff
@@ -0,0 +1,11 @@
+/* globals isFirefox */
+
```

Comment:
```javascript
const baseUrl = isFirefox ? () => window.location.href.replace(window.location.hash, '') : () => ""

Template.icon.helpers({ baseUrl })
```

Reply:
@ggazzo Why??

Reply:
So the isFirefox check run only one time.

Reply:
thanks @karlprieb

Reply:
The `if` will run only one time, but replace (much heavy) will continue running every time.

I don't see much performance improvement for such non readable code.

Reply:
That function will run over 50 times on every route change, thats why we shouldn't check if is firefox every time.